### ISSUE TITLE
Reasonable URL encoding expectations

### ISSUE THREAD

**Issue by bbabic** [a85/POSTMan-Chrome-Extension#8912]
Please, **always** fully URL encode query parameter values, no matter what is contained inside.

Why am I writing this? Because I think that it is quite reasonable to expect different behavior than current one (v7.30.1 at the moment of writing this).

Problems:
- character + (plus sign) does not get URL encoded to its respective %2b value
- %12 sequence of characters does not get encoded into %2512

Yes, I've read few issues where people are arguing that such characters should not be URL encoded per some RFC, but... you can not expect people to **type in URL encoded values in input fields**. And, if they do type something that looks like URL encoded value, you should still URL encode it, just like any other web page does since (pretty much) begginging of WWW.

My prefered solution is: fully URL encode input field. Don't over think it, just do it.

Acceptable solution: add a checkbox that would enable/disable URL encoding for input field, so that people who likes to type %xx codes can be happy.

My current environment:
Postman: v7.30.1
OS: Windows 7
HTTP Server: Netty v4.1.5 runnig on JRE7

**Comment by Boltmerz** [comment]
I'm also seeing the issue of `+` being converted to space. This issue appeared in previous builds and fixed in previous builds.

**Comment by Camios** [comment]
Agreed anything typed into the VALUE column of the Query params table should be URL encoded. I am seeing that '+' isn't being encoded too.
